src/test/shell/bazel/jdeps_test.sh

```diff
@@ -72,7 +72,7 @@ function test_jdeps() {
   # src/test/shell/bazel/jdeps_class_denylist.txt.
   find . -type f -iname \*class | \
     grep -vFf "$denylist" | \
```

Comment:
adding a `sort -r` after this should workaround the problem (for now). I'll look into a principled fix later.